### ISSUE TITLE
Suggestion: Disable drops pickup for unarmed

### ISSUE THREAD

**Issue by x1p** [JEFF-Media-GbR/Spigot-Drop2Inventory#38]
I have players that like to level for mcMMO's unarmed.
If you don't know what this is: it gives XP for killing mobs empty handed.

When players automatically pickup the drops, they soon have their hotbar full with drops and thus their hands are not empty anymore. Then unarmed no longer works. So they need to place the received item somewhere else in their inventory to empty their hand.

I now gave them the possibility to turn drop2inv off. But I'd rather not. When all players do this, the whole idea of drop2inv is bypassed.

I would be nice if the players would not pickup drops when mcmmo is installed and they kill bare handed.
Not all servers have mcmmo or want this. I myself have several servers with and without mcmmo. So a check for mcmmo, or an option would be most ideal.

**Comment by mfnalex** [comment]
Hm well... wouldn't players pick up the drops anyway since they are standing directly next to the mob? I might add an option for Drop2Inventory to only put stuff into the hotbar when the normal inventory is full if that helps?

**Comment by x1p** [comment]
That would be awesome 👍 

> wouldn't players pick up the drops anyway since they are standing directly next to the mob?

Most skyblockers use a mob tower with hoppers underneath it

**Comment by mfnalex** [comment]
Feature is ready, I'll send you a snapshot version so you can test it everything works as you would like it :)

**Comment by mfnalex** [comment]
http://repo.jeff-media.de/maven2/de/jeff_media/Drop2Inventory/4.8.0-SNAPSHOT/Drop2Inventory-4.8.0-20210102.201055-1.jar

Use this, restart server, change "avoid-hotbar" to true and restart again.

**Comment by x1p** [comment]
Works sometimes, sometimes not

**Comment by mfnalex** [comment]
strange. Can you please turn on the debug mode in your Drop2Inventory config.yml and try again, then send me your latest.log once it doesnt work?

**Comment by x1p** [comment]
```
[22:38:10 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:38:11 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:38:11 WARN]: [Drop2Inventory] [DEBUG] EntityDeathEvent
[22:38:11 WARN]: [Drop2Inventory] [DEBUG] Dropping contents for entity kill to player inv
[22:38:11 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:38:11 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
```
```
[22:40:35 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:35 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:36 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:37 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:37 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:38 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:39 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:39 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:40 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:41 WARN]: [Drop2Inventory] [DEBUG] EntityDamageByEntityEvent is NOT cancelled
[22:40:41 WARN]: [Drop2Inventory] [DEBUG] EntityDeathEvent
[22:40:41 WARN]: [Drop2Inventory] [DEBUG] Dropping contents for entity kill to player inv
```

**Comment by mfnalex** [comment]
that is obviously not the complete latest.log

**Comment by x1p** [comment]
[2021-01-02-5.log](https://github.com/JEFF-Media-GbR/Spigot-Drop2Inventory/files/5761492/2021-01-02-5.log)

**Comment by mfnalex** [comment]
Thank you, I will check this out now

**Comment by mfnalex** [comment]
Seems like it only works for blocks, not mobs - can you confirm this?

**Comment by x1p** [comment]
Confirmed

**Comment by mfnalex** [comment]
Fixed in 4.8.0, already released on SpigotMC
